### ISSUE TITLE
Screen reader is not announcing placeholder text when focus lands on input field.

### ISSUE THREAD

**Issue by msftedad** [telegramdesktop/tdesktop#27872]
### Steps to reproduce

1. Open Telegram App in Desktop.
2. Open URL: [Telegram: Contact @SCTejTestbot](https://t.me/SCTejTestbot) on edge.
3. SCTestbot home screen will appear.
4. Turn on Narrator.
5. Navigate to edit field enter any query.
6. Bot response will appear.
7. Observe an issue that Screen reader is not announcing placeholder text when focus lands on input field.

### Expected behaviour

Screen reader should announce placeholder text when focus lands on input field e.g. Write a message... edit blank.

### Actual behaviour

Screen reader is not announcing placeholder text when focus lands on input field.

https://github.com/telegramdesktop/tdesktop/assets/93735775/b51984a3-08d8-4399-951d-41cb5d7d80fa

### Operating system

22H2 (22621.1105)

### Version of Telegram Desktop

Telegram (4.16.4)

### Installation source

Microsoft Store

**Comment by Aokromes** [comment]
https://github.com/telegramdesktop/tdesktop/issues/476
